contribute-to-our-docs.rst

```diff
@@ -0,0 +1,39 @@
+Contribute to our docs
+======================
+
+We are committed to continuously improving our documentation and making it more useful for both ourselves and others.
+
+**We warmly welcome community contributions, suggestions, fixes, and constructive feedback.**
+
+If you feel something is unclear, wrong, or broken, please don't hesitate to reach out to us by any of these means:
+
+- Join the `#launchpad IRC channel`_ at `irc.libera.chat`_ and ask your
+  questions.
+- Ask a question: https://answers.launchpad.net/launchpad
+- Send an e-mail to feedback@launchpad.net
+
+Or go ahead and submit an issue or pull request at https://github.com/canonical/launchpad-manual.
```

Comment:
Personally, I would prefer to have people create issues in the very first place, as this feels more coherent, as the documentation is also hosted here on GitHub.

Reply:
Good point; will amend.